### PR TITLE
feat: add signature to footer

### PR DESCRIPTION
**Pull request by smeijer**
Adds your signature to the footer. On small screens, it's positioned on the right side of the social buttons. On medium and up, it's positioned below them.

### CODE REVIEW

File: app/components/footer.tsx [hunk 1]
```diff
@@ -110,25 +111,40 @@ function AboutSection() {
...
+          <img
+            className="absolute dark:block hidden w-full opacity-50"
+            src={images.kentSignatureDarkMode()}
+            alt={images.kentSignatureDarkMode.alt}
+          />
+          <img
+            className="absolute block dark:hidden w-full opacity-50"
+            src={images.kentSignatureLightMode()}
+            alt={images.kentSignatureLightMode.alt}
+          />
```
Comment (smeijer) [inline app/components/footer.tsx:146]: If we had the image in a gray shade, close to `gray-500` or `blueGray-500`, we could replace these two images with a single one.

**Comment by kentcdodds** [comment]
If you're going to work on an SVG, I'll wait to merge this until that's done 👍

**Comment by smeijer** [comment]
Sounds good. I'll give it a shot and let you know the outcome.

**Comment by smeijer** [comment]
Done. Looks sweet with the matching colors.

image

**Comment by kentcdodds** [comment]
Perfect 👏

**Pull request merged**